internal/resources/metal/project_ssh_key/resource.go

```diff
@@ -87,7 +88,7 @@ func (r *Resource) Read(
 	id := state.ID.ValueString()
 
 	// Use API client to get the current state of the resource
-	key, _, err := client.SSHKeys.Get(id, nil)
+	key, _, err := client.SSHKeysApi.FindSSHKeyById(context.Background(), id).Include(nil).Execute()
```

Comment:
We can leave out the `.Include` call entirely if there's nothing to include:

```suggestion
	key, _, err := client.SSHKeysApi.FindSSHKeyById(context.Background(), id).Execute()
```